botocore/data/emr/2009-03-31/waiters-2.json

```diff
@@ -37,6 +37,25 @@
           "expected": "TERMINATED_WITH_ERRORS"
         }
       ]
+    },
+    "ClusterTerminated": {
+      "delay": 30,
+      "operation": "DescribeCluster",
+      "maxAttempts": 600,
```

Comment:
Did you mean 60 here instead of 600?  Otherwise, this will block for up to 30 * 600 = 18000 seconds (5 hours).